Lombiq.NodeJs.Extensions/Docs/JavaScript.md

```diff
@@ -40,42 +40,64 @@ To use the `npm` scripts defined in this project, add any or all of the followin
   }
   ```
 
-- If you're using non-default paths (support for which is coming soon), then you will need to add the following entries (using the example paths above):
+- If you're using non-default paths, then you will need to add the following entries (using the example paths above):
 
   ```json
   "scripts": {
     "build": "npm run build:scripts",
-    "build:scripts": "npm explore nodejs-extensions -- npm run build:scripts:args --source=path/to/my/js-files --target=path/to/my/js-files",
+    "build:scripts": "npm explore nodejs-extensions -- pnpm run build:scripts --source=path/to/raw-js --target=path/to/js-files",
     "clean": "npm run clean:scripts",
-    "clean:scripts": "npm explore nodejs-extensions -- npm run clean:scripts:args --target=path/to/my/js-files",
+    "clean:scripts": "npm explore nodejs-extensions -- pnpm run clean:scripts --target=path/to/js-files",
     "watch": "npm run watch:scripts",
-    "watch:scripts": "npm explore nodejs-extensions -- npm run watch:scripts:args --source=path/to/my/js-files --target=path/to/my/js-files",
+    "watch:scripts": "npm explore nodejs-extensions -- pnpm run watch:scripts --source=path/to/raw-js --target=path/to/js-files",
   }
   ```
 
-To see the different configurations of default and non-default paths in action, please check out our dedicated [Samples](../../Lombiq.NodeJs.Extensions.Samples/Readme.md) [projects](../../Lombiq.NodeJs.Extensions.Samples.NuGet/Readme.md).
+To see the different configurations using default and non-default paths in action, please check out our dedicated [Samples](../../Lombiq.NodeJs.Extensions.Samples/Readme.md) [projects](../../Lombiq.NodeJs.Extensions.Samples.NuGet/Readme.md).
+
+
+## ESLint rules
+
+The rules are found in 2 files:
+- *.eslintrc.lombiq-base.js*: This file contains Lombiq overrides for the [airbnb-base](https://www.npmjs.com/package/eslint-config-airbnb-base) rules. It is located in *node_modules/nodejs-extensions/config*.
+- *.eslintrc.json*: In this file you can override the above Lombiq rules, or define your own [ESLint configuration](https://eslint.org/docs/user-guide/configuring/configuration-files) altogether.
+
+The *.eslintrc.json* file initially extends *.eslintrc.lombiq-base.js* from the Node.js Extensions `npm` package. It will automatically be created in your project during the first build. Should you prefer to use a global *.eslintrc.json* file for your whole solution, you can instruct Node.js Extensions to create that file in the location specified by the MSBuild property `<NodeJsExtensionsGlobalESLintConfigurationDirectory>`. This property is easiest added in a *Directory.Build.props* file in your solution's root directory as follows:
+
+```xml
+<NodeJsExtensionsCreateESLintConfigurationFile>$(MSBuildThisFileDirectory)</NodeJsExtensionsCreateESLintConfigurationFile>
```

Comment:
I agree, and this will happen as part of finishing OSOE-84. I've introduced these inconsistencies as improvements to the previous solution that I build for the SCSS pipeline. On the other hand, I felt that refactoring the latter was out of scope for the issue at hand.

Reply:
Does https://lombiq.atlassian.net/browse/OSOE-84 include a note about this?

Reply:
Now it does.